graph/db/models/node.go

```diff
@@ -0,0 +1,133 @@
+package models
+
+import (
+	"fmt"
+	"image/color"
+	"net"
+	"time"
+
+	"github.com/btcsuite/btcd/btcec/v2"
+	"github.com/btcsuite/btcd/btcec/v2/ecdsa"
+	"github.com/lightningnetwork/lnd/lnwire"
+)
+
+// LightningNode represents an individual vertex/node within the channel graph.
+// A node is connected to other nodes by one or more channel edges emanating
+// from it. As the graph is directed, a node will also have an incoming edge
+// attached to it for each outgoing edge.
+type LightningNode struct {
```

Comment:
nit: since we finally got here, how about we rename it to `GraphNode` so the name contains more useful context?

Reply:
agreed - but if it is ok - maybe we can do that at the end of this saga just to avoid many merge conflicts in the follow up work? 

or at least: let me wait to see if the second reviewer also thinks this is pressing to do now before I go ahead with a rename and rebase mission 😂

Reply:
think it's a nit so not that pressing - yeah let's move this to the end to avoid merge conflicts (we all know the pain💀).